src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -11,9 +11,10 @@
 
 logger = logging.getLogger(__name__)
 
-def activate_bluestore(meta, tmpfs, systemd):
+def activate_bluestore(meta, tmpfs, systemd, block_wal=False, block_db=False):
```

Comment:
shouldn't block_* be None instead of false, since they're either a string path or not specified?

Reply:
yes!